package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dominatr-grunt",
-  "version": "6.1.2",
+  "version": "7.0.0",
```

Comment:
It changes the default behavior of all SVG elements. To have it be a minor the default behavior should stay the same IMO.